api/test/unit/services/script-runner.service.spec.ts

```diff
@@ -1968,10 +1971,11 @@ describe('Testing script runner service', () => {
     expect(prisma.address.deleteMany).toHaveBeenCalledWith({
       where: {
         id: {
-          in: [id, id],
+          in: [id],
         },
       },
     });
+    expect(prisma.address.deleteMany).toHaveBeenCalledTimes;
```

Comment:
**issue:** I think this cuts off a bit early and doesn't do test

Reply:
must've done a undo and lost the call, thanks!